eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/common/MessageUtils.java

```diff
@@ -76,6 +77,14 @@ public static Package asyncMessageAck(Package in) {
         return msg;
     }
 
+    public static Package asyncCloudEvent(CloudEvent cloudEvent) {
+        Package msg = new Package();
+        msg.setHeader(new Header(Command.ASYNC_MESSAGE_TO_SERVER, 0,
+                EventMeshCommon.CLOUD_EVENTS_PROTOCOL_NAME, generateRandomString(seqLength)));
```

Comment:
the package header contains `Command`, `code`, `msg`, `seq`. The `msg` correspond with `code`，so please add the new attribute to represent the protocol not use the `msg`.

Reply:
ok